Dockerfile

```diff
@@ -6,12 +6,15 @@ WORKDIR /parity-bridges-ui
 
 RUN apk update
 RUN apk add git
+# Install python/pip
+ENV PYTHONUNBUFFERED=1
+RUN apk add --update --no-cache python3 && ln -sf python3 /usr/bin/python
+RUN python3 -m ensurepip
+RUN pip3 install --no-cache --upgrade pip setuptools
 
-# install app dependencies
-COPY package.json .
-COPY yarn.lock .
-RUN yarn
 COPY . .
```

Comment:
The `COPY` stuff should stay as-is, this is a common practice to utilize docker cache

Reply:
the problem i'm facing is that i need the path `src/configs` for making the `src/configs/chainsSetup/downloadCustomTypes.js` available, so i need more than the package.json and yarn.lock.

What's the best way to achieve this? Just copy the `src/configs` ? (i'm not an expert on this honestly)

Reply:
ok! reverted ... the prebuild script solved the issue i had.